editor/editor_node.cpp

```diff
@@ -4956,8 +4956,10 @@ String EditorNode::_get_system_info() const {
 		godot_version += " " + hash;
 	}
 
+	String display_server;
 #ifdef LINUXBSD_ENABLED
-	const String display_server = OS::get_singleton()->get_environment("XDG_SESSION_TYPE").capitalize().replace(" ", ""); // `replace` is necessary, because `capitalize` introduces a whitespace between "x" and "11".
+	// `replace` is necessary, because `capitalize` introduces a whitespace between "x" and "11".
+	display_server = OS::get_singleton()->get_environment("XDG_SESSION_TYPE").capitalize().replace(" ", "");
```

Comment:
I would avoid the name `display_server` for this and `distribution_display_server` to avoid furthering the ambiguity between the OS display server protocol and Godot's DisplayServer implementation.

Maybe `display_session_type`?